src/bootstrap/lib.rs

```diff
@@ -131,6 +131,7 @@ const EXTRA_CHECK_CFGS: &[(Option<Mode>, &'static str, Option<&[&'static str]>)]
     (Some(Mode::Std), "freebsd13", None),
     (Some(Mode::Std), "backtrace_in_libstd", None),
     /* Extra values not defined in the built-in targets yet, but used in std */
+    (Some(Mode::Std), "target_vendor", Some(&["unikraft"])),
```

Comment:
given that the target exists in-tree, this is only needed until the beta compiler includes the target, right?
if that's the case then you can add `// #[cfg(bootstrap)]` in a comment above so T-release can remove it when bumping the compiler.

Reply:
I think so, yes. I added the comment. 👍